### PR TITLE
a docker image with the tidb sysbench patch

### PR DESCRIPTION
**Pull request by gregwebs**

### CODE REVIEW

File: docker/tidb-sysbench/release.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+TAG="${1:-"latest"}"
+docker tag "pingcap/tidb-sysbench:$TAG" "gcr.io/pingcap-public/pingcap/tidb-sysbench:$TAG"
+gcloud docker -- push "gcr.io/pingcap-public/pingcap/tidb-sysbench:$TAG"
```
Comment (tennix) [inline docker/tidb-sysbench/release.sh]: We should also publish the image to DockerHub

File: docker/tidb-sysbench/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
+FROM alpine:3.9
+MAINTAINER PingCAP <cloud@pingcap.com>
+
+ENV sysbench_version 1.0.17
+
+# Install sysbench
+# gcc and mariadb-dev install shared libraries used at runtime
+RUN apk add gcc mariadb-dev
```
Comment (tennix) [inline docker/tidb-sysbench/Dockerfile]: It's better to install all the packages in a single `RUN` instruction and with `--no-cache` argument

Comment (gregwebs) [reply docker/tidb-sysbench/Dockerfile]: done now

File: docker/tidb-sysbench/release.sh (hunk 1 above)
Comment (gregwebs) [reply docker/tidb-sysbench/release.sh]: I don't have permission for that. Perhaps I can get permission to push just this image? But I am wondering if between this and uhub we should have some ability to move images between registries.

**Pull request merged**